code/src/components/LikeBtn.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-underscore-dangle */
+import React, { useState } from 'react';
+import './messageList.css';
+
+const LikeBtn = ({ thought, setNewLike }) => {
+  const [liked, setLiked] = useState(false);
+
+  // the function that enables the likes-count to update in the App-component
+  const handleNewLikeChange = () => {
+    setNewLike(true)
+  }
+
+  // The function that registers the users like to the database, it also
+  // triggers the state-hook that makes the button change color
+  const onButtonClick = (like) => {
+    like.preventDefault();
+
+    if (liked === false) {
```

Comment:
I think having liked === false as a condition for being able to post likes is making it so that you can only like each thought once per page refresh.  

Could you do something with counting likes instead of using true/false?